doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -163,30 +177,20 @@ Build the ACRN Hypervisor on Ubuntu
       $ git clone https://github.com/projectacrn/acrn-hypervisor
       $ cd acrn-hypervisor
 
-#. Switch to the v2.0 version:
+#. Switch to the v2.2 version:
 
    .. code-block:: none
 
-      $ git checkout -b v2.0 remotes/origin/release_2.0
+      $ git checkout -b v2.2 remotes/origin/release_2.2
```

Comment:
Something that I've wondered for a while now... why do we always change the name of the branch when checking it out? Is there any problem with just doing a `git checkout release_2.2`?

What I don't like about naming the local branch `v2.2` is that this can be confusing with the tags we also have in the repo which use that very format... let's be more consistent if possible.

**If** a `v2.2` tag will be created (and will always point at the latest code in the `release_2.2` branch, then we could ask them to check out that tag. But since we're asking to check out a branch, let's stick to its name.